nautobot/dcim/templates/dcim/softwareversion_retrieve.html

```diff
@@ -0,0 +1,80 @@
+{% block content_left_page %}
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Software Version</strong>
+                </div>
+                <div class="table-responsive">
+                    <table class="table table-hover panel-body attr-table">
+                        <tr>
+                            <td>Status</td>
+                            <td>
+                                {{ object.status|hyperlinked_object_with_color }}
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Version</td>
+                            <td>{{ object.version }}</td>
+                        </tr>
+                        <tr>
+                            <td>Alias</td>
+                            <td>{{ object.alias|placeholder }}</td>
+                        </tr>
+                        <tr>
+                            <td>Platform</td>
+                            <td>{{ object.platform|hyperlinked_object }}</td>
+                        </tr>
+                        <tr>
+                            <td>Release Date</td>
+                            <td>{{ object.release_date|placeholder }}</td>
+                        </tr>
+                        <tr>
+                            <td>End of Support Date</td>
+                            <td>{{ object.end_of_support_date|placeholder }}</td>
+                        </tr>
+                        <tr>
+                            <td>Long Term Support</td>
+                            <td>{{ object.long_term_support|render_boolean }}</td>
+                        </tr>
+                        <tr>
+                            <td>Pre-Release</td>
+                            <td>{{ object.pre_release|render_boolean }}</td>
+                        </tr>
+                        <tr>
+                            <td>Documentation URL</td>
+                            <td>
+                                {% if object.documentation_url %}
+                                <a href="{{ object.documentation_url }}" target="_blank">{{ object.documentation_url }}</a>
+                                {% else %}
+                                <span class="text-muted">&mdash;</span>
+                                {% endif %}
+                            </td>
+                        </tr>
+                    </table>
+                </div>
+            </div>
+{% endblock content_left_page %}
+
+{% block content_right_page %}
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <strong>Stats</strong>
+    </div>
+    <div class="panel-body">
+        <div class="col-md-4 text-center">
+            <h2><a href="{% url 'dcim:softwareimage_list' %}?software_version={{ object.pk }}" class="btn {% if object.software_images.count %}btn-primary{% else %}btn-default{% endif %} btn-lg">{{ object.software_images.count }}</a></h2>
```

Comment:
Are we incurring any unnecessary repeated calculations by evaluating `.count` twice here? Should we stash it in a variable that we can reuse as needed instead?

Reply:
Maybe with a `with` tag?